tools/scoring/README.md

```diff
@@ -1,6 +1,12 @@
 # Scoring
 
-The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various _dimensions_.
+The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various [_dimensions_](#dimensions). Broadly, its process is thus:
```

Comment:
can you point to an `F-Score` definition from the internet?

Reply:
(Will address in https://github.com/google/digitalbuildings/pull/549)

Reply:
https://github.com/google/digitalbuildings/pull/549/commits/b97384fb499100da6f85080782d2fcd1bf89d237